### PR TITLE
Fix/add some meta data

### PR DESCRIPTION
**Pull request by pfefferle**
use consistent title and added favicon

### CODE REVIEW

File: rel/avatar.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: default
-title: "Link Relation: avatar"
+title: "WebFinger Link Relation: avatar"
```
Comment (willnorris) [inline rel/avatar.md]: what's the difference between a "WebFinger Link Relation" and just a "Link Relation"?  While these happen to be defined within the webfinger.net namespace, there is nothing necessarily webfinger specific about them.  They're just link relations, right?

Comment (pfefferle) [inline rel/avatar.md]: I saw it more from the search engine perspective and not as a "namespace" thingy ;)

**Comment by pfefferle** [comment]
what about `<title>WebFinger | {{ page.title }}</title>` or `<title>WebFinger - {{ page.title }}</title>` instead

**Comment by willnorris** [comment]
interestingly, Google at least is already doing something like that by appending " - WebFinger" to the end of the title (which I've never noticed before):

https://www.google.com/webhp?sourceid=chrome-instant&ion=1&espv=2&ie=UTF-8#q=webfinger+link+relation+avatar

screen shot 2016-05-12 at 8 48 27 am

**Comment by pfefferle** [comment]
Interesting... But only for the pages that are missig webfinger in the title...

**Pull request closed without merging**